package/MDAnalysis/lib/_distopia.py

```diff
@@ -36,10 +37,22 @@
 else:
     HAS_DISTOPIA = True
 
+    # check for compatibility: currently needs to be >=0.2.0,<0.3.0 (issue
+    # #4740) No distopia.__version__ available so we have to do some probing.
+    needed_funcs = ['calc_bonds_no_box_float', 'calc_bonds_ortho_float']
+    has_distopia_020 = all([hasattr(distopia, func) for func in needed_funcs])
```

Comment:
There's no `distopia.__version__` attribute that I could check so I am explicitly probing for the functions we need.

@hmacdope is there a better way to get the version of the installed distopia package?

Reply:
^ See comment above, I will make a patch release